lib/setup.js

```diff
@@ -11,21 +11,24 @@ bluebird.config({
 });
 
 /**
- * optionally pass in an express app and/or templating engines
+ * optionally pass in an express app, templating engines, and/or providers
+ * note: if no providers are passed in, amphora doesn't protect routes
  * @param {object} [options]
+ * @param {array} [options.providers]
```

Comment:
the official JSDoc documentation seems to do it both ways, hm http://usejsdoc.org/tags-param.html